src/os/bluestore/bluestore_types.cc

```diff
@@ -668,23 +668,38 @@ void bluestore_blob_t::calc_csum(uint64_t b_off, const bufferlist& bl)
   }
 }
 
-int bluestore_blob_t::verify_csum(uint64_t b_off, const bufferlist& bl) const
+int bluestore_blob_t::verify_csum(uint64_t b_off, const bufferlist& bl,
+  int &b_bad_off) const
```

Comment:
style nit: can you make this int* instead of int& to indicate it is an output argument?